cronjobs/add_authority_external_ref.py

```diff
@@ -141,25 +141,34 @@ def Main():
                 util.ExecOrDie(util.Which("add_authority_external_ref"), [
                                "--create_mapping_file", f"/tmp/{input_file_name_for_add_auth}", f"/tmp/{gnd_wiki_file}"])
 
-                print("Process 6/7 -- Copy the gnd_wiki_file to its destination")
-                util.ExecOrDie(util.Which("cp"), [
-                               "-f", f"/tmp/{gnd_wiki_file}", f"{share_folder}{gnd_wiki_file}"])
-
-                #  e. Update the latest version date on the config file (the second line)
-                print(
-                    "Process 7/7 -- Updating the config file and remove temporary files")
-                UpdateConfigFile(
-                    config_file, current_file_date_int)
+                
                 util.ExecOrDie(util.Which("rm"), [
                                "-f", f"/tmp/{newer_file_name}"])
                 util.ExecOrDie(util.Which("rm"), [
                                "-f", f"/tmp/{input_file_name_for_add_auth}"])
+            
+                file_size = os.path.getsize(f"/tmp/{gnd_wiki_file}")
+                
+                if(file_size == 0):
+                    print("Process 6/7 -- Sending error message via email")
```

Comment:
This is not step 6/7 => If the GND file is empty, it should not be copied to the destination. The script should abort after sending the mail and print a proper message.